### PR TITLE
Remove outdated comment?

### PR DESCRIPTION
**Pull request by follower**
This advice appears to now be outdated & contain a search/replace error based on the recent change from PoolVector to Vector: https://github.com/godotengine/godot/commit/3205a92ad872f918c8322cdcd1434c231a1fd251#diff-e3b190dd79504a83af3fff42156517be

I may however be misinterpreting the advice. :)

### CODE REVIEW

File: core/vector.h [hunk 1]
```diff
@@ -34,7 +34,7 @@
 /**
  * @class Vector
  * @author Juan Linietsky
- * Vector container. Regular Vector Container. Use with care and for smaller arrays when possible. Use Vector for large arrays.
+ * Vector container. Regular Vector Container.
```
Comment (akien-mga) [inline core/vector.h]: Could you change it to:
```
Vector container that uses a Copy on Write mechanism for passing data around efficiently.
```

Comment (akien-mga) [reply core/vector.h]: (Note: I cancelled the CI builds as a change in `vector.h` means a 50 min build stalling every other PR, so I prefer to keep it only for the final approved change ;))

**Comment by aaronfranke** [comment]
@follower Can you please update the PR with the above suggested changes?

**Comment by akien-mga** [comment]
Closing as stale. As changing such a header imposes a full engine rebuild to all people compiling from Git, I prefer we make it count and add more useful documentation for the class.

**Pull request closed without merging**